docker/opbeans/node/entrypoint.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env sh
 set -ex
 if [ -f /local-install/package.json ]; then
-    echo "Installing from local folder"
-    # copy to folder inside container to ensure were not poluting the local folder
-    cp -r /local-install ~
-    cd ~/local-install && npm install .
```

Comment:
This `npm install .` was installing the deps for the apm-agent-nodejs copy in ~/local-install, but it wasn't installing them where the app at "/app" would use it.